build/repo.targets

```diff
@@ -35,22 +35,29 @@
   </Target>
 
   <Target Name="GetNativeArtifactsInfo">
+    <PropertyGroup>
+      <AncmPackageOutputPath>$(BuildDir)Microsoft.AspNetCore.AspNetCoreModule.$(PackageVersion).nupkg</AncmPackageOutputPath>
+      <AncmV2PackageOutputPath>$(BuildDir)Microsoft.AspNetCore.AspNetCoreModuleV2.$(PackageVersion).nupkg</AncmV2PackageOutputPath>
+    </PropertyGroup>
+
     <ItemGroup Condition=" '$(OS)' == 'Windows_NT' ">
-      <ArtifactInfo Include="$(BuildDir)Microsoft.AspNetCore.AspNetCoreModule.$(PackageVersion).nupkg">
+      <ArtifactInfo Include="$(AncmPackageOutputPath)">
         <ArtifactType>NuGetPackage</ArtifactType>
         <PackageId>Microsoft.AspNetCore.AspNetCoreModule</PackageId>
         <Version>$(PackageVersion)</Version>
         <RepositoryRoot>$(RepositoryRoot)</RepositoryRoot>
       </ArtifactInfo>
-      <FilesToExcludeFromSigning Include="$(BuildDir)Microsoft.AspNetCore.AspNetCoreModule.$(PackageVersion).nupkg" />
+      <FilesToSign Include="$(AncmPackageOutputPath)" Certificate="$(PackageSigningCertName)" />
+      <FilesToSign Include="$(RepositoryRoot)tools\installancm.ps1" Container="$(AncmPackageOutputPath)" Certificate="$(PowerShellSigningCertName)" />
 
-      <ArtifactInfo Include="$(BuildDir)Microsoft.AspNetCore.AspNetCoreModuleV2.$(PackageVersion).nupkg">
+      <ArtifactInfo Include="$(AncmV2PackageOutputPath)">
         <ArtifactType>NuGetPackage</ArtifactType>
         <PackageId>Microsoft.AspNetCore.AspNetCoreModuleV2</PackageId>
         <Version>$(PackageVersion)</Version>
         <RepositoryRoot>$(RepositoryRoot)</RepositoryRoot>
       </ArtifactInfo>
-      <FilesToExcludeFromSigning Include="$(BuildDir)Microsoft.AspNetCore.AspNetCoreModuleV2.$(PackageVersion).nupkg" />
+      <FilesToSign Include="$(AncmV2PackageOutputPath)" Certificate="$(PackageSigningCertName)" />
+      <FilesToSign Include="$(RepositoryRoot)tools\installancm.ps1" Container="$(AncmV2PackageOutputPath)" Certificate="$(PowerShellSigningCertName)" />
```

Comment:
You can remove this file; I don't believe it is used anywhere.

Reply:
It's in the package, so it needs to be signed. Can we remove the file from the package?

https://github.com/aspnet/IISIntegration/blob/804fc191f648f00e0256e6d62b26f116bc53e5e3/nuget/Microsoft.AspNetCore.AspNetCoreModuleV2.nuspec#L41

Reply:
Yes.

Reply:
Done.